### PR TITLE
Include EMR scripts in generate-tarball

### PR DESCRIPTION
**Pull request by ns1123**
Include EMR scripts in generate-tarball
Fix docs to provide more information on bootstrap arguments

### CODE REVIEW

File: docs/en/compute/AWS-EMR.md [hunk 1]
```diff
@@ -34,29 +34,53 @@ the root UFS can be reconfigured to be HDFS.
...
+5. Test that Alluxio is running as expected by running `sudo runuser -l alluxio -c "/opt/alluxio/bin/alluxio runTests"`
 
 Alluxio is installed in `/opt/alluxio/` by default. Hive and Presto are already configured to connect to Alluxio. The
 cluster also uses AWS Glue as the default metastore for both Presto and Hive. This will allow you to maintain table
 definitions between multiple runs of the Alluxio cluster.
 
+See the below sample command for reference.
+
+```bash
+aws emr create-cluster --release-label emr-5.23.0 --instance-count 3 --instance-type m4.xlarge --applications Name=Presto Name=Hive --name 'Test cluster' --bootstrap-actions Path=s3://alluxio-test/emr/bootstrap-actions/alluxio-emr.sh,Args=[http://downloads.alluxio.io/downloads/files/2.0.0-preview/alluxio-2.0.0-preview-bin.tar.gz,s3a://alluxio-test/emr/mount/,alluxio.underfs.s3.owner.id.to.username.mapping=f1234123412341234123412341234123412341234123412341234123412341234hadoop] --configurations file:///Users/foo/emr/alluxio/alluxio-emr.json --ec2-attributes KeyName=admin-key
```
Comment (rastogiasr) [inline docs/en/compute/AWS-EMR.md]: owner.id.to.username.mapping is missing '=' before hadoop.

File: docs/en/compute/AWS-EMR.md [hunk 2]
```diff
@@ -34,29 +34,53 @@ the root UFS can be reconfigured to be HDFS.
...
+aws emr create-cluster --release-label emr-5.23.0 --instance-count <num-instances> --instance-type <instance-type> --applications Name=Presto Name=Hive Name=Spark --name '<cluster-name>' --bootstrap-actions Path=s3://bucket/path/to/alluxio-emr.sh,Args=[<download-url>,<root-ufs-uri>,<additional-properties>,<license-url>] --configurations file:///path/to/file/alluxio-emr.json --ec2-attributes KeyName=<ec2-keypair-name>
 ```
+<!-- ALLUXIO CS END -->
 
-3. On the [EMR Console](https://console.aws.amazon.com/elasticmapreduce/home), you should be able to see the cluster
+4. On the [EMR Console](https://console.aws.amazon.com/elasticmapreduce/home), you should be able to see the cluster
 going through the different stages of setup. Once the cluster is in the 'Waiting' stage, click on the cluster details
 to get the 'Master public DNS'. SSH into this instance using the keypair provided in the previous command.
-4. Test that Alluxio is running as expected by running `sudo runuser -l alluxio -c "/opt/alluxio/bin/alluxio runTests"`
+5. Test that Alluxio is running as expected by running `sudo runuser -l alluxio -c "/opt/alluxio/bin/alluxio runTests"`
```
Comment (rastogiasr) [inline docs/en/compute/AWS-EMR.md]: Add the IP inbound rule comment before this line.

**Comment by madanadit** [comment]
alluxio-bot, merge this please.

**Comment by madanadit** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**